src/components/skeletons/Shimmer/shimmer.scss

```
@@ -0,0 +1,24 @@
+[data-store-shimmer] {
+  /* parent: position relative & overflow hidden */
+  position: absolute;
+
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  animation: loading 1.2s infinite;
+
+  [data-shimmer] {
+    width: 50%;
+    height: 100%;
+    background: rgb(255 255 255 / 20%);
+    transform: skewX(-20deg);
+    box-shadow: 0 0 var(--space-5) var(--space-5) rgb(255 255 255 / 20%);
```

```suggestion
    box-shadow: 0 0 var(--space-5) var(--space-5) rgb(255 255 255 / 20%);
    transform: skewX(-20deg);
```
Stylelint complains about this order